openssl-dynamic/src/main/c/sslcontext.c

```diff
@@ -1655,6 +1655,7 @@ static int cert_requested(SSL* ssl, X509** x509Out, EVP_PKEY** pkeyOut) {
             break;
     }
 #else
+    (void) ssl2_ctype;
```

Comment:
> Option 2 won't work because we take the address of it, and it's allocated on the stack.

Good point ... its used else where i see.

> Option 1 works, but my version is overall less code.

After compile or before compile :wink:. I'm ok with the current approach ... however if the issue we are trying to resolve is "get rid of a warning because we don't use the variable" ... instead of artificially using it to appease the warning ... why not just "fix" the warning and get rid of the variable if its not needed (ifdef it out)?